### ISSUE TITLE
Comments on latest PR

### ISSUE THREAD

**Issue by timothyas** [ECCO-GROUP/ECCOv4-py#116]
1. A small comment, can we default to deleting code rather than commenting it out? This reduces the clutter, and it's not necessary because git keeps track of the changes for us :) e.g.
https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/get_basin.py#L22-L23
https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/get_basin.py#L76
https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/get_basin.py#L82

2. Another small one, since we're using `os.path.join` we might as well use it all the way: e.g. replace
https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/get_basin.py#L25
with 
`basin_path = os.path.join(package_directory, '..', 'binary_data'),`

3. I don't really understand why the following lines are necessary. Does it have to do with running the tests in parallel? If they're just debug statements then I think they should be removed:
https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/test/test_common.py#L33-L39

https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/test/test_common.py#L71-L77

https://github.com/ECCO-GROUP/ECCOv4-py/blob/83e2f219a1e491cf30cfd285c9170d04ec6d1014/ecco_v4_py/test/test_common.py#L89-L95

**Comment by ifenty** [comment]
@timothyas 
1. Yes let's delete old (dead, commented) lines
2. sure
3. it reduces the number of times the mds files need to be downloaded.  Old behavior was that files were redownloaded dozens of times (maybe even every time setup_mds_dir() was called).  New behavior is that is downloads about one order of magnitude fewer.  I would love to get it down to less (once?) but I haven't gotten that far yet.  The global variables point to the directories where the mds files are downloaded.  If `mds_dir_info == None`, the mds files are downloaded.  If not none, the directory with the downloaded mds files is returned.

Does that make sense?

I'll make new PR with (1) and (2)

**Comment by timothyas** [comment]
Oh cool, that's great. Thanks @ifenty
